src/app/public/modules/select-field/select-field-picker.component.html

```diff
@@ -10,6 +10,11 @@
           [selectedIds]="selectedIds"
           (selectedIdsChange)="onSelectedIdsChange($event)">
           <sky-list-toolbar>
+              <sky-list-toolbar-search-actions  >
+                  <button *ngIf="showNewButton" class="sky-btn sky-btn-default sky-select-field-picker-btn-new" (click)="emitAddNewRecord()" >
```

Comment:
Let's move the `*ngIf="showNewButton"` to the `<sky-list-toolbar-search-actions>` tag so that it won't show an empty region if the button isn't being used.

```
<sky-list-toolbar-search-actions *ngIf="showNewButton">
```